Add some channels

BellezaEmporium opened the pull request.

BellezaEmporium committed:
Add some Anime channels.

LaneSh4d0w commented:
Will need to verify https://ani-box.com/

LaneSh4d0w commented:
@Dum4G Try this channel http://ott-cdn.ucom.am/s81/03.m3u8 with your known User-Agent VLC fix, and tell me if you get Setanta Sports 1.

Dum4G commented:
> Will need to verify https://ani-box.com/

![image](https://user-images.githubusercontent.com/5430116/127785934-dcc68ad8-4708-497a-bd61-d7394fc9fa95.png)

Dum4G commented on channels/br.m3u:
```diff
@@ -5,6 +5,8 @@ http://45.162.230.234:1935/agrobrasiltv/agrobrasiltv/playlist.m3u8
 https://5cf4a2c2512a2.streamlock.net/dgrau/dgrau/playlist.m3u8
 #EXTINF:-1 tvg-id="Animestation.br" tvg-name="Animestation" tvg-country="BR" tvg-language="" tvg-logo="https://i.imgur.com/5UpjGcL.png" group-title="",Animestation (480p) [Not 24/7]
 https://stmv.video.expressolider.com.br/animestation1/animestation1/playlist.m3u8
+#EXTINF:-1 tvg-id="AnimeTV.br" tvg-name="AnimeTV" tvg-country="BR" tvg-language="" tvg-logo="" group-title="",Anime TV
```
Animation category

LaneSh4d0w commented:
> 
> 
> > Will need to verify https://ani-box.com/
> 
> ![image](https://user-images.githubusercontent.com/5430116/127785934-dcc68ad8-4708-497a-bd61-d7394fc9fa95.png)

Yep, I suspect it being a not 24/7 channel, despite the fact that the link hasn't changed.

BellezaEmporium committed:
Update br.m3u

BellezaEmporium committed:
Update ru.m3u

BellezaEmporium committed:
Update ru.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Add channels from the Hungama Digital Network

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Take off the Shemaroo channels

reason : Channels aren't updated since 2019-2020 and streams are just 5 seconds of a movie then cuts off.

Dum4G commented on channels/ru.m3u:
```diff
@@ -880,7 +880,7 @@ http://1hdru-hls-otcnet.cdnvideo.ru/siesta/tracks-v1a1/index.m3u8
 #EXTINF:-1 tvg-id="SmaylikTV.ru" tvg-name="Смайлик ТВ" tvg-country="RU" tvg-language="Russian" tvg-logo="http://smilik.tv/wp-content/uploads/2016/02/Desktop_-1.png" group-title="Kids",Смайлик ТВ (720p)
 http://62.32.67.187:1935/WEB_Smilik/Smilik.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="SportTV.ru" tvg-name="Sport TV" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Sports",Sport ТV [No Sound]
+#EXTINF:-1 tvg-id="SportTV.ru" tvg-name="Sport TV" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Sports",Sport ТV
```
What a mess, where is it from? Almost convinced myself it's a Polish channel but now it shows a single Ukrainian ad 7 times in a row. Absolutely unsigned channel

LaneSh4d0w replied:
Weird... found it in a random playlist lurking on the Internet.
What's interesting is that they gave him 3 different names... cool to find the actual name (Fight Box, Sport TV, UFC...)

Dum4G replied:
That's getting funnier. Referee speaks in Russian and commentary sounds like an alien

LaneSh4d0w replied:
![vlcsnap-2021-08-02-01h31m55s682](https://user-images.githubusercontent.com/30985701/127788509-2e9590dc-7bd8-469c-85c3-88ea12f7e576.png)

LaneSh4d0w replied:
> 
> 
> That's getting funnier. Referee speaks in Russian and commentary sounds like an alien

Lol ! At that state, i'm gonna remove that channel, since we can't seem to put our hands on what it is x)

Dum4G replied:
It's 1+1 Спорт from Ukraine, only available here
https://1plus1.video/tvguide/sport/online

LaneSh4d0w replied:
![image](https://user-images.githubusercontent.com/30985701/127788689-9ca813ca-dc96-44fb-a4fc-c8e5dec79f91.png)
Found that out, we already had it.

LaneSh4d0w replied:
Seems they got different servers, https://live-fdc07-ams.1plus1.video/live/1627959900/IP2NXOS92aRcPdJ1tUd_hg/sport/smil:sport.smil/playlist.m3u8 is another one of them, given by https://grandcentral.1plus1.video/live/36/3b1b37e842d182c884ef55cd8951cc30/1627959900?return_http=true&_t744940799313

Dum4G replied:
That's definitely not НТВ+ as it ceased to exist years ago

LaneSh4d0w replied:
Interesting...

Well, we shall have a look again at some of the Russian channels then. You'll sort them better than I, I suppose lol

BellezaEmporium committed:
Update ru.m3u

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

LaneSh4d0w commented:
I guess that'll be enough for now.

Dum4G commented on channels/in.m3u:
```diff
@@ -400,68 +410,38 @@ http://158.69.124.9:1935/sardaritv/sardaritv/playlist.m3u8
 #EXTINF:-1 tvg-id="ServeurTV.in" tvg-name="Serveur TV" tvg-country="IN" tvg-language="English" tvg-logo="" group-title="",Serveur TV
 https://a.jsrdn.com/broadcast/060753d37e/+0000/high/c.m3u8
-#EXTINF:-1 tvg-id="ShemarooBollywoodClassic.in" tvg-name="Shemaroo Bollywood Classic" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/cMJVuIr.jpg" group-title="Movies",Shemaroo Bollywood Classic
-https://livechannel.shemaroome.com/linearplayout/02-bollywood-classic/chunklist_1920x1080_cf.m3u8
```
Source works. Was it a part of some cleanup?

LaneSh4d0w replied:
Yes. It's the same chunk of program over and over again. If you look at the root URL you'll see files that weren't updated since 2019/2020.
